packages/embed-jsx/package.json

```diff
@@ -0,0 +1,42 @@
+{
+	"name": "@sapphire/embed-jsx",
+	"version": "1.0.0",
+	"description": "An easy way to use JSX to create discord embeds with discord.js",
+	"keywords": [
+		"embed",
+		"discord",
+		"bot",
+		"typescript",
+		"ts",
+		"yarn",
+		"sapphire"
+	],
+	"author": "Gryffon Bellish <owenbellish@gmail.com>",
```

Comment:
Please change to match other packages. We give credit through all contributors and if you so choose we can also attribute this part of the code to you as code owners.